### PR TITLE
Revive Skia Native Sample

### PR DESCRIPTION
**Pull request by alexander-gorshenev**

### CODE REVIEW

File: skiko/src/commonMain/kotlin/org/jetbrains/skia/Path.kt [hunk 1]
```diff
@@ -435,7 +430,8 @@ class Path internal constructor(ptr: NativePointer) : Managed(ptr, _FinalizerHol
             t: Float,
             r: Float,
             b: Float,
-            radii: FloatArray?,
+            radii: InteropPointer,
+            size: Int,
```
Comment (olonho) [inline skiko/src/commonMain/kotlin/org/jetbrains/skia/Path.kt:434]: radiiSize

File: skiko/src/jvmMain/cpp/common/Canvas.cc [hunk 2]
```diff
@@ -245,7 +245,7 @@ extern "C" JNIEXPORT void JNICALL Java_org_jetbrains_skia_Canvas__1nClipRect
 }
 
 extern "C" JNIEXPORT void JNICALL Java_org_jetbrains_skia_Canvas__1nClipRRect
-  (JNIEnv* env, jclass jclass, jlong canvasPtr, jfloat left, jfloat top, jfloat right, jfloat bottom, jfloatArray jradii, jint mode, jboolean antiAlias) {
+  (JNIEnv* env, jclass jclass, jlong canvasPtr, jfloat left, jfloat top, jfloat right, jfloat bottom, jfloatArray jradii, jint size, jint mode, jboolean antiAlias) {
```
Comment (olonho) [inline skiko/src/jvmMain/cpp/common/Canvas.cc:248]: radiiSize

File: skiko/src/macosX64Main/cpp/generated/Canvas.cc [hunk 3]
```diff
@@ -80,29 +81,19 @@ extern "C" void org_jetbrains_skia_Canvas__1nDrawOval
...
-
-
 
 extern "C" void org_jetbrains_skia_Canvas__1nDrawDRRect
   (kref __Kinstance, jlong canvasPtr,
    jfloat ol, jfloat ot, jfloat oright, jfloat ob, jfloatArray ojradii,
    jfloat il, jfloat it, jfloat ir, jfloat ib, jfloatArray ijradii,
    jlong paintPtr) {
-    throw std::runtime_error("TODO: implement org_jetbrains_skia_Canvas__1nDrawDRRect");
+    TODO("implement org_jetbrains_skia_Canvas__1nDrawDRRect");
```
Comment (olonho) [inline skiko/src/macosX64Main/cpp/generated/Canvas.cc:96]: Do we skip those just to avoid commonization?

**Review by olonho — COMMENTED** [review]
Looks good.

File: skiko/src/macosX64Main/cpp/common/BackendRenderTarget.cc [hunk 4]
```diff
@@ -36,7 +36,7 @@ extern "C" jlong BackendRenderTarget_nMakeMetal
 #endif
 
 extern "C" jlong BackendRenderTarget_MakeDirect3D
-  (jint width, jint height, jlong texturePtr, jint format, jint sampleCnt, jint levelCnt) {
+  (kref __Kinstance, jint width, jint height, jlong texturePtr, jint format, jint sampleCnt, jint levelCnt) {
```
[the remark this answers is not in the document]
Comment (alexander-gorshenev) [reply skiko/src/macosX64Main/cpp/common/BackendRenderTarget.cc:39]: Yes. The `external` kotlin function is still in a companion object.  
We'll move `external` fuctions to the global scope with a separate fix.

File: skiko/build.gradle.kts [hunk 5]
```diff
@@ -399,11 +399,12 @@ project.tasks.register<Exec>("nativeBridgesCompile") {
         *targetOs.clangFlags,
         "-std=c++17",
         "-c",
+        "-g",
```
[the remark this answers is not in the document]
Comment (alexander-gorshenev) [reply skiko/build.gradle.kts]: utilized the  buildType

File: skiko/src/jvmMain/cpp/common/Canvas.cc [hunk 6]
```diff
@@ -70,7 +70,7 @@ extern "C" JNIEXPORT void JNICALL Java_org_jetbrains_skia_Canvas__1nDrawOval
 }
 
 extern "C" JNIEXPORT void JNICALL Java_org_jetbrains_skia_Canvas__1nDrawRRect
-  (JNIEnv* env, jclass jclass, jlong canvasPtr, jfloat left, jfloat top, jfloat right, jfloat bottom, jfloatArray jradii, jlong paintPtr) {
+  (JNIEnv* env, jclass jclass, jlong canvasPtr, jfloat left, jfloat top, jfloat right, jfloat bottom, jfloatArray jradii, jint size, jlong paintPtr) {
```
[the remark this answers is not in the document]
Comment (alexander-gorshenev) [reply skiko/src/jvmMain/cpp/common/Canvas.cc]: renamed in this place and other places

File: skiko/src/macosX64Main/cpp/generated/Canvas.cc (hunk 3 above)
Comment (alexander-gorshenev) [reply skiko/src/macosX64Main/cpp/generated/Canvas.cc]: Ah, I didn't realize there was only of left using skija::RRect::toSkiaRRect. Converted this one too.

**Review by olonho — APPROVED** [review]

**Pull request merged**